static/styles/lib/bootstrap/scss/_forms.scss

```diff
@@ -171,6 +171,10 @@ select.form-control-sm {
   padding: $input-padding-y-lg $input-padding-x-lg;
   font-size: $font-size-lg;
   @include border-radius($input-border-radius-lg);
+  -moz-hyphens: none; 
```

Comment:
You don't need to add vendor prefixes here. Gulp will do that for you so you only need one line.